rest/models/src/main/java/com/connexta/store/rest/models/MetadataInfo.java

```diff
@@ -0,0 +1,95 @@
+package com.connexta.store.rest.models;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.swagger.annotations.ApiModelProperty;
+import java.net.URL;
+import java.util.Objects;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+/** Information representing Metadata. */
+public class MetadataInfo {
+
+  @JsonProperty("metadataType")
+  private String metadataType;
+
+  @JsonProperty("location")
+  private URL location;
+
+  public MetadataInfo metadataType(String metadataType) {
+    this.metadataType = metadataType;
+    return this;
+  }
+
+  @ApiModelProperty(example = "irm", required = true, value = "The type of metadata.")
+  @NotNull
+  @Size(min = 1, max = 80)
+  public String getMetadataType() {
+    return metadataType;
+  }
+
+  public void setMetadataType(String metadataType) {
+    this.metadataType = metadataType;
+  }
+
+  public MetadataInfo location(URL location) {
+    this.location = location;
+    return this;
+  }
+
+  /**
+   * The URL for where the metadata is located.
+   *
+   * @return location
+   */
+  @ApiModelProperty(
+      example = "https://www.example.com/mis/30f14c6c1fc85cba12bfd093aa8f90e3",
```

Comment:
✏️ We don't use the path `/mis` anymore. A better example is `http://store:9040/dataset/30f14c6c1fc85cba12bfd093aa8f90e3/irm`.